src/types.ts

```diff
@@ -507,4 +512,12 @@ export type AppCreditInfo = {
   burnedSupply: number,
   maxSupply: number | null,
   createdAt: number,
-};
\ No newline at end of file
+};
+
+export type ActivityNftInfo = {
+  chain: string;
+  network: string;
+  contractAddress: string;
+  tokenId: string;
+  userAddress: string;
+}
```

Comment:
Can you add a new line at the end of the file?

Reply:
done. Thanks!